<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TestsWidget</class>
 <widget class="QWidget" name="TestsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>327</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="layoutDirection">
   <enum>Qt::LeftToRight</enum>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="opaqueResize">
      <bool>false</bool>
     </property>
     <property name="handleWidth">
      <number>5</number>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <property name="spacing" stdset="0">
      <number>2</number>
     </property>
     <property name="leftMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="topMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="rightMargin" stdset="0">
      <number>2</number>
     </property>
     <property name="bottomMargin" stdset="0">
      <number>2</number>
     </property>
     <widget class="QWidget" name="tabDescription">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <property name="topMargin">
        <number>0</number>
       </property>
       <property name="rightMargin">
        <number>0</number>
       </property>
       <property name="bottomMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QSplitter" name="splitterLeft">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="handleWidth">
          <number>4</number>
         </property>
         <property name="childrenCollapsible">
          <bool>false</bool>
         </property>
         <widget class="QWidget" name="horizontalLayoutWidget">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>1</number>
           </property>
           <item>
            <widget class="QWidget" name="widgetLeftSide" native="true">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>1</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="leftMargin">
               <number>0</number>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <property name="rightMargin">
               <number>0</number>
              </property>
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QToolBar" name="toolBarTests">
                <property name="floatable">
                 <bool>false</bool>
                </property>
                <addaction name="actionAddTest"/>
                <addaction name="actionRemoveTest"/>
               </widget>
              </item>
              <item>
               <widget class="QListWidget" name="listWidgetTests"/>
              </item>
             </layout>
            </widget>
           </item>
           <item>
            <widget class="QTabWidget" name="tabWidget">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
               <horstretch>3</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="tabPosition">
              <enum>QTabWidget::South</enum>
             </property>
             <property name="currentIndex">
              <number>0</number>
             </property>
             <widget class="QWidget" name="tabSteps">
              <attribute name="title">
               <string>Steps</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_4">
               <property name="spacing">
                <number>0</number>
               </property>
               <property name="leftMargin">
                <number>0</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>0</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QToolBar" name="toolBarSteps">
                 <property name="floatable">
                  <bool>false</bool>
                 </property>
                 <addaction name="actionAddStep"/>
                 <addaction name="actionRemoveStep"/>
                 <addaction name="actionRecordSteps"/>
                 <addaction name="actionRunSteps"/>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listWidgetSteps">
                 <property name="dragDropMode">
                  <enum>QAbstractItemView::InternalMove</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabSettings">
              <attribute name="title">
               <string>Settings</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_5">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>0</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_2">
                 <property name="text">
                  <string>Starting Action</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ActionSelectWidget" name="actionSelect" native="true"/>
               </item>
               <item>
                <widget class="QLabel" name="label">
                 <property name="text">
                  <string>Starting Inventory</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QListWidget" name="listInventory"/>
               </item>
               <item>
                <widget class="QToolBar" name="toolBar">
                 <property name="floatable">
                  <bool>false</bool>
                 </property>
                 <addaction name="actionAddObject"/>
                 <addaction name="actionRemoveObject"/>
                </widget>
               </item>
              </layout>
             </widget>
             <widget class="QWidget" name="tabScripts">
              <attribute name="title">
               <string>Scripts</string>
              </attribute>
              <layout class="QVBoxLayout" name="verticalLayout_6">
               <property name="spacing">
                <number>3</number>
               </property>
               <property name="leftMargin">
                <number>8</number>
               </property>
               <property name="topMargin">
                <number>3</number>
               </property>
               <property name="rightMargin">
                <number>8</number>
               </property>
               <property name="bottomMargin">
                <number>0</number>
               </property>
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="text">
                  <string>Initialization</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScriptEdit" name="scriptEditInit"/>
               </item>
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="text">
                  <string>Validity Check (runs after test)</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="ScriptEdit" name="scriptEditCheck">
                 <property name="placeholderText">
                  <string>return true;</string>
                 </property>
                </widget>
               </item>
              </layout>
             </widget>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidgetRight">
      <layout class="QVBoxLayout" name="verticalLayoutRight">
       <property name="spacing">
        <number>1</number>
       </property>
       <item>
        <widget class="NovelTeaWidget" name="preview" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>200</width>
           <height>300</height>
          </size>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
  <action name="actionAddTest">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Test</string>
   </property>
  </action>
  <action name="actionRemoveTest">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove Test</string>
   </property>
  </action>
  <action name="actionAddStep">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Step</string>
   </property>
  </action>
  <action name="actionRemoveStep">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove Step</string>
   </property>
  </action>
  <action name="actionAddStepAction">
   <property name="text">
    <string>Action</string>
   </property>
  </action>
  <action name="actionAddStepWait">
   <property name="text">
    <string>Wait</string>
   </property>
  </action>
  <action name="actionRunSteps">
   <property name="icon">
    <iconset theme="media-playback-start">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Run Steps</string>
   </property>
  </action>
  <action name="actionRecordSteps">
   <property name="icon">
    <iconset theme="media-record">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Record Steps</string>
   </property>
  </action>
  <action name="actionAddStepDialogueOption">
   <property name="text">
    <string>Dialogue Option</string>
   </property>
  </action>
  <action name="actionAddObject">
   <property name="icon">
    <iconset theme="list-add">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Add Object</string>
   </property>
  </action>
  <action name="actionRemoveObject">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset theme="list-remove">
     <normaloff/>
    </iconset>
   </property>
   <property name="text">
    <string>Remove Object</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>NovelTeaWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/NovelTeaWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ActionSelectWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/ActionSelectWidget.hpp</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>ScriptEdit</class>
   <extends>QPlainTextEdit</extends>
   <header>Widgets/ScriptEdit.hpp</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
